graphql/handler/transport/websocket.go

```diff
@@ -44,7 +44,7 @@ type (
 		initPayload InitPayload
 	}
 
-	WebsocketInitFunc  func(ctx context.Context, initPayload InitPayload) (context.Context, error)
+	WebsocketInitFunc  func(ctx context.Context, initPayload InitPayload) (*InitPayload, context.Context, error)
```

Comment:
feel free to resolve, but what about keeping the same ordering here for the params and the return?

Reply:
Done